reposerver/src/main/scala/com/advancedtelematic/tuf/reposerver/db/Repository.scala

```diff
@@ -155,8 +158,9 @@ protected[db] class SignedRoleRepository()(implicit val db: Database, val ec: Ex
         .failIfNone(SignedRoleNotFound)
     }
 
-  def storeAll(targetItemRepo: TargetItemRepository)(signedRoles: List[SignedRole], items: Seq[TargetItem]): Future[Unit] = db.run {
-    DBIO.sequence(signedRoles.map(sr => persistAction(sr, forceVersion = false)))
+  def storeAll(targetItemRepo: TargetItemRepository)(repoId: RepoId, signedRoles: List[SignedRole], items: Seq[TargetItem]): Future[Unit] = db.run {
+    targetItemRepo.resetAction(repoId)
+      .andThen(DBIO.sequence(signedRoles.map(sr => persistAction(sr, forceVersion = false))))
```

Comment:
what is this for?

Reply:
what do you mean? this is what makes the test pass.